.github/workflows/ddev-playwright.yml

```diff
@@ -79,6 +87,23 @@ jobs:
             JSON
               .parse(process.env.ENV_VARS)
               .forEach(envVar => core.exportVariable(envVar.name, envVar.value));
+            
+      - name: Setup SSH keys
+        run: |
+            mkdir -p .ddev/homeadditions/.ssh
+            echo "${{ secrets.SSH_KEY }}" > .ddev/homeadditions/.ssh/id_rsa
+            echo "StrictHostKeyChecking=accept-new" > .ddev/homeadditions/.ssh/config
+            echo "CheckHostIP=no" > .ddev/homeadditions/.ssh/config
+            echo "HashKnownHosts=no" >> .ddev/homeadditions/.ssh/config
+            chmod 700 .ddev/homeadditions/.ssh
+            chmod 600 .ddev/homeadditions/.ssh/id_rsa
+            chmod 600 .ddev/homeadditions/.ssh/config
+        if: ${{ env.SSH_KEY != '' }}
+
+      - name: Setup SSH knows hosts
+        run: |
+          echo "${{ secrets.SSH_KNOWN_HOSTS }}" > .ddev/homeadditions/.ssh/known_hosts
+        if: ${{ env.SSH_KNOWN_HOSTS != '' }}
```

Comment:
Same here; please move below `name`.
Ah, I see this order is used at other steps, too. I'd really appreciate it if you could align everything according to the Pathfinder rules. Thanks!